framework/utils.py

```diff
@@ -32,6 +32,32 @@ def is_internal_ip(ip):
     parsed_ip = IPAddress(ip)
     return parsed_ip.is_private
 
+def get_rank(user_rank, owtf_rank):
```

Comment:
Are you sure that such function is not already available? Either in OWTF or even in PTP?